jenkinsfile-runner/Dockerfile

```diff
@@ -1,5 +1,9 @@
+# Inhereting form 'latest' Jenkins master is ok since this image is controlled by us
+# hadolint ignore=DL3006
 FROM ppiper/jenkins-master
 
+# User needs to be root for access to mounted Docker socket
+# hadolint ignore=DL3002
 USER root
```

Comment:
According to https://github.com/hadolint/hadolint/wiki/DL3002 do we need to switch back to a non-root user after installation and copying?